### PR TITLE
feat(output): convert ANSI escape sequences to HTML elements formatting

### PR DESCRIPTION
**Pull request by miltolstoy**
Sources:
```
\u001b[31m red \u001b[0m, \u001b[42m\u001b[34m blue on green \u001b[0m, \u001b[1m\u001b[4m\u001b[36m bold underline cyan \u001b[0m
```
Console: 
Screenshot from 2021-12-15 07-39-35
HTML:
```
<span style="color: #aa0000">red</span>
<span style="color: #0000aa; background-color: #00aa00">blue on green</span>
<span style="font-weight: bold; text-decoration: underline; color: #00aaaa">bold underline cyan</span>
```
Screenshot from 2021-12-15 07-41-57

### CODE REVIEW

File: universum/modules/output/html_output.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 import re
 from datetime import datetime
 
+from ansi2html import Ansi2HTMLConverter
```
Comment (i-keliukh) [inline universum/modules/output/html_output.py:5]: Conditional import has to be done in different way. See perforce module for example.

File: universum/modules/output/html_output.py [hunk 2]
```diff
@@ -130,6 +135,23 @@ def _build_html_head(self):
...
+        if not self.ansi_style_converter:
+            return line
+        return self.ansi_style_converter.convert(line, full=False)
+
+    @staticmethod
+    def _try_retrieve_ansi_converter():
+        class AnsiModuleStub(ModuleType):
+            Ansi2HTMLConverter: Callable
+        try:
+            ansi_module = cast(AnsiModuleStub, importlib.import_module("ansi2html"))
```
Comment (i-keliukh) [inline universum/modules/output/html_output.py:148]: Why you are importing the module this way if it is a mandatory requirement now?

File: universum/modules/output/html_output.py [hunk 3]
```diff
@@ -1,7 +1,10 @@
+import importlib
```
Comment (i-keliukh) [inline universum/modules/output/html_output.py:1]: This import is not needed anymore

File: universum/modules/output/html_output.py [hunk 4]
```diff
@@ -147,3 +151,8 @@ def _wrap_links(line):
     def _build_time_stamp():
         now = datetime.now()
         return now.astimezone().strftime('<span class="time" title="%Z (UTC%z)">%Y-%m-%d %H:%M:%S</span> ')
+
+    @staticmethod
+    def _ansi_codes_to_html(line):
+        converter = Ansi2HTMLConverter(inline=True, escaped=False)
```
Comment (i-keliukh) [inline universum/modules/output/html_output.py:157]: Please create converter instance in the constructor

**Review by i-keliukh — APPROVED** [review]

**Comment by k-dovgan** [comment]
Looks good to me.

**Pull request merged**